custom_components/tahoma/cover.py

```diff
@@ -19,10 +19,11 @@
     SUPPORT_SET_TILT_POSITION,
     SUPPORT_STOP,
     SUPPORT_STOP_TILT,
-    CoverEntity,
+    CoverDevice,
```

Comment:
Isn't `*Device` deprecated in favor of `*Entity`

`CoverDevice is deprecated, modify to extend CoverEntity
11:00:49 AM – Cover (WARNING)`

Reply:
Ah yes! But it's weird my ide only find CoverDevice not CoverEntity